tests/foreman/ui/test_docker.py

```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so we cannot fetch information about it
+    @stubbed('Unstub when BZ1266842 is fixed')
+    @run_only_on('sat')
+    @tier2
+    def test_verify_bugzilla_1266842(self):
```

Comment:
I think we should use a meaningful name. As I commented here https://github.com/SatelliteQE/robottelo/pull/3304#discussion_r53657092, eventually the BZ skip will be removed and having a test that requires checking the docstring or the related BZ is not good as one with a meaningful name that by just looking on jenkins you can get information about the test.